consensus.go

```diff
@@ -244,6 +263,17 @@ func (p *Pbft) Run(ctx context.Context) {
 	}
 }
 
+func (p *Pbft) RunPrepare(ctx context.Context) {
```

Comment:
Since this method is making initial state of the state machine, maybe we can rename it?